subprojects/core-api/src/main/java/org/gradle/api/model/ObjectFactory.java

```diff
@@ -119,7 +120,7 @@
     <T> ListProperty<T> listProperty(Class<T> elementType);
 
     /**
-     * Creates a {@link SetProperty} implementation to hold a {@link Set} of the given element type {@code T}. The property has no initial value.
+     * Creates a {@link SetProperty} implementation to hold a {@link Set} of the given element type {@code T}. The property has an empty list as its initial value.
```

Comment:
This is a set, but you document it as:
> The property has an empty list as its initial value.

Shouldn't this say:

> The property has an empty set as its initial value.

Reply:
Resolved here: https://github.com/gradle/gradle/pull/7969

Reply:
Thanks for fixing that.